cli/tools/repl/session.rs

```diff
@@ -528,18 +528,16 @@ impl ReplSession {
       .post_message_with_event_loop(
         "Runtime.callFunctionOn",
         Some(cdp::CallFunctionOnArgs {
-          function_declaration: format!(
-            r#"function (object) {{ {}.lastEvalResult = object; }}"#,
-            *REPL_INTERNALS_NAME
-          ),
-          object_id: None,
+          function_declaration: r#"function (object) { this.lastEvalResult = object; }"#.to_string(),
+          object_id: REPL_INTERNAL_OBJECT_ID.with_borrow(Clone::clone),
           arguments: Some(vec![evaluate_result.into()]),
           silent: None,
           return_by_value: None,
           generate_preview: None,
           user_gesture: None,
           await_promise: None,
-          execution_context_id: Some(self.context_id),
+          // execution_context_id: Some(self.context_id),
+          execution_context_id: None,
```

Comment:
Ditto

Reply:
@CyanChanges I still don't understand why this is `None` now - I assume because we are now setting `object_id`. Is this correct? Can you please remove the commented out lines?